index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Quiz</title>
+	<link rel="stylesheet" type="text/css" href="reset.css">
+	<link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+	<div id="container">
+		<article id="product1" class="productCard">
+			<header class="productHeader">
+				<h1>Adorable Kitten</h1>
+			</header>
+			<section class="productInfo1">
```

Comment:
**Pro tip:** Instead of naming classes in numerical sequence, name them to reflect the content that will be in the corresponding HTML element. If your product owner or designer wants to change the order of how the information is displayed, then it reduces confusion among the other developers on your team because they won't see `productInfo3`, `productInfo1`, then `productInfo2` when the order changes.